tycho-core/src/main/java/org/eclipse/tycho/core/TargetPlatformConfiguration.java

```diff
@@ -156,11 +156,7 @@ public void setPomDependencies(PomDependencies pomDependencies) {
 
     public PomDependencies getPomDependencies() {
         if (pomDependencies == null) {
-            if (isRequireEagerResolve()) {
-                return PomDependencies.ignore;
-            } else {
-                return PomDependencies.consider;
-            }
+            return PomDependencies.consider;
```

Comment:
This is an incompatible change, please the the documentation, consider is only the default for eager resolve is not configured.